docker-compose.yml

```diff
@@ -26,4 +26,4 @@ services:
     environment:
       - PASSWORD=test
       - TAGS=[]
-      - PACKAGES=[]
+      - PACKAGES=["scipy", "numpy"]

```

Comment:
can we make this just `PACKAGES` ans use .env file?

Reply:
sure